JapaneseFilter/sections/specific.txt

```diff
@@ -99,6 +99,15 @@ weblio.jp##.adBannerHdWrp
 weblio.jp##.adSmpBnrFtBtSp
 weblio.jp##.adBkOverlayClr
 new-akiba.com###bannerB2
+iza.ne.jp##.header-link > ul > li > a[href="http://www.sankei-digital.co.jp/"]
+||prt.iza.ne.jp/common/css/v2/images/bg_header_003.png^$domain=iza.ne.jp
```

Comment:
Request not reproduced - also $domain is unnecessary.
Same for other url block rule.